config/pmd/ruleset.xml

```diff
@@ -66,9 +66,11 @@
   </rule>
   <rule ref="category/java/errorprone.xml">
     <exclude name="NonSerializableClass"/>
+    <exclude name="MissingSerialVersionUID"/>
   </rule>
   <rule ref="category/java/multithreading.xml"/>
   <rule ref="category/java/performance.xml"/>
   <rule ref="category/java/security.xml"/>
   <exclude-pattern>.*/generated.*</exclude-pattern>
+  <exclude-pattern>.*/controller/.*</exclude-pattern>
```

Comment:
because PMD was complaining about the way that openApiGnerate was generating the controller class. Its now been decided we dont need one for this PR, so i can remove it again, but it might come back in the future

Reply:
removed